roles/cinder-common/templates/etc/cinder/rootwrap.conf

```diff
@@ -10,11 +10,7 @@ filters_path=/etc/cinder/rootwrap.d,/usr/share/cinder/rootwrap
 # explicitely specify a full path (separated by ',')
 # If not specified, defaults to system PATH environment variable.
 # These directories MUST all be only writeable by root !
-{% if openstack_install_method == "package" -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,{{ 'cinder'|ursula_package_path(openstack_package_version) }}/bin
-{% else -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin
-{% endif -%}
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/opt/openstack/current/cinder/bin
```

Comment:
what happens for distro method where there is no /op/openstack/current/cinder/bin ?